mlir/test/Dialect/Vector/vector-shape-cast-lowering-scalable-vectors.mlir

```diff
@@ -7,7 +7,7 @@
 // CHECK-SAME: %[[arg0:.*]]: vector<2x1x[4]xi32>
 func.func @i32_3d_to_1d_last_dim_scalable(%arg0: vector<2x1x[4]xi32>) -> vector<[8]xi32>
 {
-  // CHECK-NEXT: %[[cst:.*]] = arith.constant dense<0> : vector<[8]xi32>
+  // CHECK-NEXT: %[[cst:.*]] = ub.poison : vector<[8]xi32>
```

Comment:
[nit] Here and other places - since this is no longer an  `arith.constant`, perhaps let's rename the LIT variables? `cst` is just going to be confusing (and most likely copied by people).

`cst` -> `poison`? `cst` -> `init` ? `cst` -> `ubp`? `cst` -> `undef`.

Naming is hard 🤷🏻‍♂️ Avoiding `cst` should be a "win" regardless of the alternative.